CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,9 @@
+# OpenPubkey Community Code of Conduct
+
+OpenPubkey follows the [GitHub Community Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct). 
```

Comment:
I don't think we can adopt this as-is. This CoC is for the GitHub Community product which is the discussion boards at https://github.com/orgs/community/discussions. The copy of the CoC refers to GitHub Community throughout and suggests contacting GitHub staff in case of a violation.

I suggest we adopt [Contributor Covenant](https://www.contributor-covenant.org/) instead. The pledge is pretty similar.

/cc @EthanHeilman, I know you had thoughts on this

Reply:
I'd be good with adopting the Contributor Covenant. Can we put the full text in this file?

Reply:
Revised: [d5b3a69](https://github.com/openpubkey/community/pull/1/commits/d5b3a6956668937ac10bdf095be50d889cc51cad)